### DISCUSSION TITLE
How to install OpenDB correctly ?

### DISCUSSION THREAD

**Discussion by b224hisl** [The-OpenROAD-Project/OpenROAD#3540] [Q&A] [answered]
Excuse me, I'm installing the OpenDB(https://github.com/The-OpenROAD-Project/OpenDB), following the installation guide like this:
```
git clone https://github.com/The-OpenROAD-Project/OpenDB.git 
cd OpenDB 
mkdir build
cd build
cmake ..
make
```

But I met some problem when executing `cmake..`:
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39954153/054e75ce-9e89-4443-887a-0a81cb99b912)
Does anyone knows how to handle this problem?

**Comment by maliberty** [comment]
You need to install spdlog.  The standalone OpenDB is not getting active development if that matters.  Changes are made to the integrated version in OpenRoad.

**Reply by b224hisl** [reply]
but after I installed the spdlog, it still has some problems shown like this:
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/39954153/8b15f644-f2a3-4809-a3be-cd9bab95ab73)
Do you know how to handle it?

**Reply by b224hisl** [reply, marked as answer]
I have handled this problem, just install tcl-dev. Anyway thanks a lot!
